README.md

```diff
@@ -223,3 +223,81 @@ var routes = [
    }
 ];
 ```
+##### Specifying policies dynamically as functions
+
+In the `config.plugins.policies` array you can also include raw policy functions.
+```javascript
+var isAdminPolicy = function isAdmin (request, reply, next) {
+    
+    if (hasAdminAccess(request)) {
+        next(null, true);
+    } else {
+        next(null, false)
+    }
+};
+
+isAdminPolicy.applyPoint = 'onPreHandler';
+
+var routes = [
+   {
+       method: 'your_method',
+       path: '/your/path/here',
+       handler: your_route_handler,
+       config: {
+           plugins: {
+               policies: [ isAdminPolicy ]
+           }
+       }
+   }
+];
+```
+
+##### Running policies in parallel
+If you'd like to run policies in parallel, you can specify a list of loaded policies as an array or as individual arguments to `MrHorse.parallel`.
+When policies are run in parallel, expect all policies to complete.  If any of the policies specify an error or `Forbidden 403` message, by default the first listed policy with such an error will be given precedence.
+
+```javascript
+var routes = [
+   {
+       method: 'your_method',
+       path: '/your/path/here',
+       handler: your_route_handler,
+       config: {
+           plugins: {
+               policies: [
+                    ['isAdmin', 'smellsGood']
```

Comment:
Can you add another fake policy in there to highlight the fact that there are two arrays in play?  Like:
```
[
'isInUS',
['isAdmin', 'smellsGood']
]
```